.github/workflows/_01_pre_check.yml

```diff
@@ -61,6 +61,9 @@ jobs:
       - name: Checkout chainflip-backend
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
 
+      - name: Install pnpm
```

Comment:
Seems like this broke the upgrade-test on the release branch: https://github.com/chainflip-io/chainflip-backend/actions/runs/8392274235/job/22984548822